scripts/compare_benchmarks.py

```diff
@@ -64,8 +64,11 @@ def calculate_and_format_p_value(old, new):
     return colored(notes + "{0:.4f}".format(p_value), color)
 
 
-if(len(sys.argv) != 3):
-    exit("Usage: " + sys.argv[0] + " benchmark1.json benchmark2.json")
+if(len(sys.argv) != 3 and len(sys.argv) != 4):
+    exit("Usage: " + sys.argv[0] + " benchmark1.json benchmark2.json [--github]")
+
+# Format the output as a diff (prepending - and +) so that Github shows colors
+github_format = bool(len(sys.argv) == 4 and sys.argv[3] == '--github')
```

Comment:
Not really nice, but I did not want to add full argument parsing just for this. We could also simply make this the default - maybe people can live with ` ```diff ` prepended and `+`/`-` in front of the table.

Reply:
I would leave it as it is now.